include/mbedtls/cipher.h

```diff
@@ -507,6 +507,77 @@ static inline const char *mbedtls_cipher_info_get_name(
         return( info->MBEDTLS_PRIVATE(name) );
 }
 
+/**
+ * \brief       This function returns the size of the IV or nonce
+ *              for the cipher info structure, in bytes.
+ *
+ * \param info  The cipher info structure. This may be \c NULL.
+ *
+ * \return      The recommended IV size.
+ * \return      \c 0 for ciphers not using an IV or a nonce.
+ */
+static inline int mbedtls_cipher_info_get_iv_size(
+    const mbedtls_cipher_info_t *info )
+{
+    if( info == NULL )
+        return( 0 );
+
+    return( (int) info->MBEDTLS_PRIVATE(iv_size) );
+}
+
+/**
+ * \brief        This function returns the block size of the given
+ *               cipher info structure.
+ *
+ * \param info   The cipher info structure. This may be \c NULL.
+ *
+ * \return       The block size of the cipher.
```

Comment:
Please document the unit (both bits and bytes are plausible).

Please document the value for ciphers that aren't block ciphers.

Reply:
I updated the documentation for both `mbedtls_cipher_info_get_block_size` and `mbedtls_cipher_get_block_size`.